Containerfile

```diff
@@ -37,6 +37,10 @@ COPY --from=cache /build/ /usr/src/shuttle/
 FROM shuttle-common
 ARG folder
 COPY ${folder}/prepare.sh /prepare.sh
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protoc-21.9-linux-x86_64.zip &&\
```

Comment:
The prepare.sh install of shuttle-runtime complained errored on not having `protoc` installed, but it should be installed. Is there a better way to do this?